src/components/base.astro

```diff
@@ -8,7 +8,7 @@
 		<meta name="theme-color" media="(prefers-color-scheme: light)" content="#ECECEC">
 		<meta name="theme-color" media="(prefers-color-scheme: dark)"  content="#222">
 		<meta name="robots" content="noindex">
-		<title>Notebook Archive</title>
+		<slot name="head"/>
```

Comment:
I really like how Nuxt will automatically scoop up `<head>` tags. This is sort of close to that.